packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -92,66 +94,69 @@ Cons:
 
 ### API
 
-#### Icon column element:
-
-_Component Name_
-
--   `nimble-table-column-icon`
-
-_Props/Attrs_
-
--   `field-name`: string
--   `key-type`: 'string' | 'number' | 'boolean'
-
-_Content_
-
--   column title (icon)
--   1 or more `nimble-mapping-icon` or `nimble-mapping-spinner` elements
-
-#### General mapping column element:
+#### Enum column element:
 
 _Component Name_
 
--   `nimble-table-column-enum-text`
+-   `nimble-table-column-enum`
```

Comment:
I may have missed a discussion thread or offline chat, but why `nimble-table-column-enum` instead of `nimble-table-column-mapping`? In the [table column naming guidance](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/table/specs/table-columns-hld.md#column-type-philosophy) we talk about `nimble-table-column-mapping` because the presentation of the column is a mapping element. Using `-enum` seems more  like data type terminology than presentation terminology to me, which we haven't done so far.

If this has already been debated an there's good rationale for `enum` then I don't want to prolong the discussion. But we should update that naming guidance to include this new example and remove that note about mapping columns.

Reply:
We did discuss this in a meeting and settled on `nimble-table-column-enum`, but I don't actually remember why we preferred `-enum` rather than `-mapping`. @rajsite, do you remember? If I'm not mistaken, I think you were the one that suggested `-enum`.

Reply:
I think I said that because since then we had renamed the column `enum` from `mapping` so seemed like just a stale doc.

I don't think that on their own either 'enum' or 'mapping' clearly represent presentations. Also, mappings at least initially were intended as generic models hence not owning a template / specific visual and components like enum / icon column or rich-text mention decide how to render specific mappings (and may do so differently).

If we say mappings are visual models (still don't own a specific view but are describing parameters of any(?) presentation of them) then I guess `nimble-table-column-mapping` is a better name since mappings are closer to presentations in that point-of-view. I think that makes adding things like "text-hidden" seem conceptually more aligned where before it was feeling maybe too table view specific.

Under that mental model I'm aligned on `nimble-table-column-mapping`